frontend/src/component/layout/MainLayout/NavigationSidebar/useRoutes.ts

```diff
@@ -9,41 +9,57 @@ import {
 import { useInstanceStatus } from 'hooks/api/getters/useInstanceStatus/useInstanceStatus';
 import type { INavigationMenuItem } from 'interfaces/route';
 import type { IUiConfig } from 'interfaces/uiConfig';
+import { useUiFlag } from 'hooks/useUiFlag';
+import { useVariant } from 'hooks/useVariant';
+import type { Variant } from 'utils/variants';
 
-const filterRoutes = (
-    routes: INavigationMenuItem[],
-    uiConfig: IUiConfig,
-    { enterprise, pro, billing }: PlanData,
-) => {
-    return routes
-        .filter(filterByConfig(uiConfig))
-        .filter((route) =>
-            filterRoutesByPlanData(route?.menu, {
-                enterprise,
-                pro,
-                billing,
-            }),
-        )
-        .map(normalizeRoutePath);
+const useNewRoute = () => {
+    const flag = useUiFlag('newInUnleash');
+    return useVariant(flag as Variant);
```

Comment:
This is the flag control. It might not be the right final solution, but because it defaults to false, I think it's safe to merge like this anyway.